.github/actions/start-local-server/action.yml

```diff
@@ -27,7 +27,7 @@ runs:
         # Give it some time before starting to poll.
         sleep 10
         started=1
-        for i in `seq 1 50`;
+        for i in `seq 1 120`;
```

Comment:
Poking through various workflow runs, this consistently times out but we never see it because there is no exit() call. I've added one here and bumped this to a more generous timeout. I am unsure, but I would not be surprised if this contributed to some test instability in the past.